resources/careconnect-to-ukcore/allergyintolerance/input/AllergyIntoleranceDescriptionId-Extension-3to4_001.json

```diff
@@ -0,0 +1,33 @@
+{
+  "resourceType": "AllergyIntolerance",
+  "code": {
+    "coding": [
+      {
+        "extension": [
+          {
+            "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-CodingSCTDescId",
```

Comment:
As this is the input this url should be

https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid

or

https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-coding-sctdescid

as that is all that is handled in the map at the momemt